### PR TITLE
AN-67 create models intially

### PR DESCRIPTION
**Pull request by nurlanmemmedov**

### CODE REVIEW

File: src/main/java/com/example/turboaz/enums/GearBox.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.example.turboaz.enums;
+
+public enum GearBox {
+    AUTOMATIC,
+    MECHANICAL,
+    ROBOTISED,
+    VARIATOR;
```
Comment (mubarizmammadzada) [inline src/main/java/com/example/turboaz/enums/GearBox.java]: son iki gearboxlar-dan chox da istifade olunmur axi

File: src/main/java/com/example/turboaz/models/Make.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.example.turboaz.models;
+
+public class Make {
+    private int id;
+    private String name;
+}
```
Comment (mubarizmammadzada) [inline src/main/java/com/example/turboaz/models/Make.java:9]: Make-de list<model> saxlamali deyilik?

File: src/main/java/com/example/turboaz/enums/BodyType.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.example.turboaz.enums;
+
+public enum BodyType {
+    CARAVAN,
+    SEDAN,
```
Comment (kamaltaghiyev) [inline src/main/java/com/example/turboaz/enums/BodyType.java:5]: We need in BodyType also bus, moto cycle, jeep, hatchback and crossover

**Review by kamaltaghiyev — COMMENTED** [review]
In b

**Review by kamaltaghiyev — APPROVED** [review]

**Review by mubarizmammadzada — APPROVED** [review]

**Pull request merged**